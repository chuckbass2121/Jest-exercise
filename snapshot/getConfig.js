export function getConfig() {
	return {
		url: 'abc',
		env: 'dev',
		createdAt: new Date(),
	};
}
