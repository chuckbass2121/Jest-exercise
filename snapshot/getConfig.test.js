import { getConfig } from './getConfig';

test('should getConfig', () => {
	expect(getConfig()).toMatchSnapshot({
		createdAt: expect.any(Date),
	});
});

test('should getConfig inline', () => {
	expect(getConfig()).toMatchInlineSnapshot(
		{
			createdAt: expect.any(Date),
		},
		`
    Object {
      "createdAt": Any<Date>,
      "env": "dev",
      "url": "abc",
    }
  `
	);
});
