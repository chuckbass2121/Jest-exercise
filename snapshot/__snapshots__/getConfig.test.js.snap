// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should getConfig 1`] = `
Object {
  "createdAt": Any<Date>,
  "env": "dev",
  "url": "abc",
}
`;
